src/main/java/com/bobocode/bibernate/session/SessionImpl.java

```diff
@@ -28,15 +33,24 @@ public class SessionImpl implements Session {
 
     private final EntityPersister entityPersister;
 
-    public SessionImpl(EntityPersister entityPersister, Dialect dialect) {
+    private final PersistenceContext persistenceContext;
+
+    public SessionImpl(EntityPersister entityPersister, Dialect dialect, PersistenceContext persistenceContext) {
```

Comment:
I don`t think we should pass `PersistenceContext` as parameter of `session`, `PersistenceContext` is like a "helper mechanism" incapsulated inside the session so I think we can create in in the constructor by `new`